testutil/exec.go

```diff
@@ -79,3 +82,14 @@ func (cmd *MockCmd) Calls() []string {
 	text = strings.TrimSuffix(text, "\n")
 	return strings.Split(text, "\n")
 }
+
+// ForgetCalls purges the list of calls made so far
+func (cmd *MockCmd) ForgetCalls() {
```

Comment:
Only nitpicking here. In python mock its called reset_mock(), maybe simply `Reset()` ?